config/config.go

```diff
@@ -25,11 +25,24 @@ var (
 	pathConfig       = cue.MakePath(cue.Str("config"))
 )
 
-// Load loads the configuration at the given directory and returns it.
+type options struct {
+	tags []string
+}
+
+type optionFunc func(o *options)
+
+// WithTags returns an option function to provide a list of tag values when loading the configuration.
+func WithTags(tags []string) optionFunc {
```

Comment:
Modulo the possibility of removing this entirely (see below), I'd generally suggest always returning an exported type from an exported function.

Reply:
It's going to go away per the earlier comment.

Reply:
Done